### ISSUE TITLE
Make a package

### ISSUE THREAD

**Issue by scls19fr** [saulpw/visidata#2]
Hello,

you should make a package for visidata

Kind regards

**Comment by saulpw** [comment]
Thanks for the suggestion.  I would like to keep VisiData light and hackable, so the less overhead the better.  It's not a big deal, and since someone else requested this too, I probably will make a package for it (at least by v1.0).  But I don't really understand, what is the advantage to making a package for it, when the script can just be copied as-is to a bin directory?

**Comment by scls19fr** [comment]
see https://github.com/firecat53/tabview/blob/master/setup.py or https://github.com/wavexx/gtabview/blob/master/setup.py

with 

    scripts=['bin/tabview'],

your script will be automatically stored in a bin directory which is part of your PATH so it will be easier to install it and have it available from any directory.

**Comment by TMiguelT** [comment]
The main advantage, I imagine, is that you could just `pip install visidata` instead of
```bash
git clone https://github.com/saulpw/visidata
cd visidata
cp vd.py /usr/local/bin
```

**Comment by saulpw** [comment]
Okay, I merged a setup.py into master with v0.30.  Can you test that it installs properly for you?

**Comment by scls19fr** [comment]
You might enable Travis and have a very basic test to ensure that it installs correctly

**Comment by TMiguelT** [comment]
Thanks for making a package. Can you start publishing versions?

**Comment by saulpw** [comment]
Yes, I am finishing up some work on it this week, and will publish the next version on pypi very soon.  Thanks for pinging me on this.

**Comment by saulpw** [comment]
I published v0.37 on pypi tonight.

**Comment by TMiguelT** [comment]
Many thanks! I look forward to to trying it out!
